src/pages/_proxied-dot-io/waypoint/index.tsx

```diff
@@ -8,99 +11,47 @@ import MonitorAndManage from 'components/_proxied-dot-io/waypoint/homepage/secti
 import s from './home/style.module.css'
 
-function HomePage(): JSX.Element {
+function HomePage({ data }): JSX.Element {
+	const {
+		seo,
+		hero,
+		intro,
+		configureYourAppSection,
+		buildAndDeploySection,
+		monitorYourAppSection,
+		monitorAppHealthSection,
+		extendPluginsSection,
+		callToActionSection,
+		footer,
+	} = data
+
 	return (
-		<div className={s.homePage}>
-			<Hero
-				heading={
-					<>
-						Easy application deployment for <em>Kubernetes</em> and{' '}
-						<em>Amazon ECS</em>
-					</>
-				}
-				description="Waypoint allows developers to deploy, manage, and observe their applications through a consistent abstraction of underlying infrastructure. Waypoint works with Kubernetes, ECS and many other platforms."
-				link={{
-					title: 'Get Started',
-					url: 'https://learn.hashicorp.com/collections/waypoint/get-started-kubernetes',
-				}}
-			/>
-			<SectionIntro
-				columnLeft={{
-					heading: (
-						<>
-							Simple <em>developer experience</em>
-						</>
-					),
-					description:
-						'Waypoint provides a simple and consistent abstraction for developers to easily build, deploy, and release applications.',
-					features: [
-						{
-							icon: require('components/_proxied-dot-io/waypoint/homepage/icons/layers.svg?include'),
-							heading: 'Application-centric abstraction',
-							description:
-								'Specify the deployment needs with a simple and consistent abstraction without the underlying complexity.',
-						},
-						{
-							icon: require('components/_proxied-dot-io/waypoint/homepage/icons/link.svg?include'),
-							heading: 'End-to-end deployment workflow',
-							description:
-								'Move and manage resources efficiently with distinct build, deploy, release steps.',
-							link: {
-								text: 'Learn more',
-								url: 'https://www.waypointproject.io/docs/lifecycle',
-							},
-						},
-					],
-				}}
-				columnRight={{
-					heading: (
-						<>
-							Powerful for <em>operators</em>
-						</>
-					),
-					description:
-						'Waypoint enables operators to create PaaS workflows of Kubernetes, ECS, serverless applications.',
-					features: [
-						{
-							icon: require('components/_proxied-dot-io/waypoint/homepage/icons/maximize.svg?include'),
-							heading: 'Build-deploy-release extensibility',
-							description:
-								'Enable a pluggable framework, integrated with CI/CD pipelines, monitoring tools, and any other ecosystem tools.',
-							link: {
-								text: 'Learn more',
-								url: 'https://www.waypointproject.io/docs/extending-waypoint',
-							},
-						},
-						{
-							icon: require('components/_proxied-dot-io/waypoint/homepage/icons/sidebar.svg?include'),
-							heading: 'PaaS experience for developers',
-							description:
-								'Provide a consistent abstraction and unified workflow for any major platforms.',
-						},
-					],
-				}}
+		<WaypointIoLayout footer={footer}>
```

Comment:
I opted to wrap the page with the Waypoint layout as opposed to using `HomePage.layout = WaypointIoLayout` to set the layout.  This is because `<Min100Layout />` expects footer to be passed into it, and we're fetching the data at the page level.